README.md

```diff
@@ -1,31 +1,34 @@
+Mayastor incorporates Intel's [Storage Performance Development Kit](https://spdk.io/). It has been designed from the ground up to leverage the protocol and compute efficiency of NVMe-oF semantics, and the performance capabilities of the latest generation of solid-state storage devices, in order to deliver a storage abstraction with performance overhead measured to be within the range of single-digit percentages.
 
-By comparison most pre-CAS shared everything storage systems are widely thought to impart an overhead of at least 40% and sometimes as much as 80% or more as compared to the capabilities of the underlying devices or cloud volumes; additionally pre-CAS shared storage scales in an unpredictale manner as I/O from many workloads interact and compete for the capabilities of the shared storage system.
+By comparison, most "shared everything" storage systems are widely thought to impart an overhead of at least 40% (and sometimes as much as 80% or more) as compared to the capabilities of the underlying devices or cloud volumes; additionally traditional shared storage scales in an unpredictale manner as I/O from many workloads interact and compete for resources.
 
-While Mayastor utilizes NVMe-oF it does not require NVMe devices or cloud volumes to operate.
+While Mayastor utilizes NVMe-oF it does not require NVMe devices or cloud volumes to operate and can work well with other device types.
 
 ## Where can I find Mayastor?
 
-The Mayastor product is associated with multiple repositories under the OpenEBS project. The list of repositories are as follows:
-- [openebs/mayastor](https://github.com/openebs/mayastor) : This repository contains specifications for Mayastor Data Plane components.
-- [openebs/mayastor-control-plane](https://github.com/openebs/mayastor-control-plane) : This repository contains specifications for Mayastor Control Plane components.
-- [openebs/mayastor-api](https://github.com/openebs/mayastor-api) : This repository contains protocol buffer definitions and OpenAPI specifications for Mayastor
+Mayastor's source code and documentation are distributed amongst a number of GitHub repositories under the OpenEBS organisation.  The following list describes some of the main repositories but is not exhaustive. 
+
+- [openebs/mayastor](https://github.com/openebs/mayastor) : contains the source code of the data plane components
+- [openebs/mayastor-control-plane](https://github.com/openebs/mayastor-control-plane) : contains the source code of the control plane components
+- [openebs/mayastor-api](https://github.com/openebs/mayastor-api) : contains common protocol buffer definitions and OpenAPI specifications for Mayastor components
+- [openebs/mayastor-dependencies]:(https://github.com/openebs/mayastor-dependencies): contains dependencies common to the control and data plane repositories
```

Comment:
Colon not needed after ]
Space needed after )

Reply:
Resolved: 3ca46b8